frontend/src/components/pages/UserActions/ActivityDashBoardBarGraph/ActivityDashboardBarGraph.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import activityLogBarGraphStorybookData from "./storyData/activityLogBarGraphStorybookData.json";
+import { ActivityDashboardBarGraphUnconnected } from ".";
+
+export default {
+  title: "Tilde/pages/userActions/ActivityDashboardBarGraph",
+  component: ActivityDashboardBarGraphUnconnected,
+};
+
+const Template = (args) => <ActivityDashboardBarGraphUnconnected {...args} />;
+
+export const Primary = Template.bind({});
+
+Primary.args = {
+  activityLogDayCounts: activityLogBarGraphStorybookData.activityLogDayCount,
```

Comment:
Why not have consistent naming? Why is it a pleural sometimes and a singular sometimes? Someone's going to make a bug when they wire this up and then someone is going to have to fix it.